sources/packages/backend/apps/queue-consumers/src/services/workflow/workflow-enqueuer.service.ts

```diff
@@ -58,6 +68,40 @@ export class WorkflowEnqueuerService {
     }
   }
 
+  /**
+   * Search applications with assessments to be cancelled.
+   * If no other assessment is being cancelled for that application the oldest
```

Comment:
`If no other assessment is being cancelled` ? I am not able to follow the comment.

Reply:
Change the comment.